src/common/tse_internal.h

```diff
@@ -55,7 +55,9 @@ struct tse_task_private {
 					 dtp_completing:1,
 					/* task is in running state */
 					 dtp_running:1,
-					 dtp_dep_cnt:29;
+					/* Don't propagate err-code from dependent tasks */
+					 dtp_dont_propagate:1,
```

Comment:
minor nit, perhaps no_propagate.  dont without the ' hurts my eyes

other possibilities:
avoid_propagation
disable_propagate

Or perhaps an opposite
suppress_error
censor_error

Reply:
ok, will change if need refresh the patch later or by following patch.